code/modules/clothing/spacesuits/rig/suits/combat.dm

```diff
@@ -52,8 +52,9 @@
 
 
 	initial_modules = list(
-		/obj/item/rig_module/vision/thermal,
+		//obj/item/rig_module/vision/thermal, //no
```

Comment:
It would best to not comment out lines. They are instead removed out.

Reply:
Agreed. No reason to keep in dead lines like that.